### DISCUSSION TITLE
建议增加书签分类和时间排序

### DISCUSSION THREAD

**Discussion by SwampH** [alefragnani/vscode-bookmarks#723] [Ideas]
建议增加书签分类和时间排序

**Comment by alefragnani** [comment]
Hi @SwampH ,

Could you please translate to English? I can use some translator, but I had bad experiences / feedback when doing so on a good will, so I prefer avoid it for now.

Thank you

**Reply by SwampH** [reply]
I really like this plugin, and I have some suggestions for adding new features to it:
1. Bookmark classification function: It means that it can be classified according to the project code module
2. Time sorting function: can be arranged in ascending or descending order based on time

**Reply by alefragnani** [reply]
Hi,

About the "bookmark classification", I'm not sure what you mean by _according to the project code module_ but maybe #254 would work.  About the "time sorting function", I guess #183 would fit your needs. Otherwise, a new option to #217

If those open issues are not what you expect, feel free to open new issues, providing more details about it. Better yet, if you add some use case on how the idea would be used, in order to better understand the requirements.

Hope this helps
